config/initializers/assets.rb

```diff
@@ -3,9 +3,10 @@
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
 
-Rails.application.config.assets.precompile += %w( home.jpg )
-Rails.application.config.assets.precompile += %w( musikore.svg )
-Rails.application.config.assets.precompile += %w( musikore-com.svg )
+Rails.application.config.assets.precompile += %w(home.jpg)
```

Comment:
If should be fine if you added the images to the images folder. You should have restarted your server to precompile the assets.